tools/pipelines/templates/build-npm-package.yml

```diff
@@ -166,6 +166,7 @@ stages:
       # Job - Build
       - job: build
         displayName: Build
+        timeoutInMinutes: 90
```

Comment:
not sure if you can use template parameters to conditionally set timeoutInMinutes. Assuming you can (which I think is true), doing so for only CI would also be an option if we care. This change is simpler though and the PR build for it should help validate it does correctly update the timeout.

Reply:
Yeah, a template-expression-syntax conditional here should work if we wanted to.